packages/terra-heading/tests/nightwatch/HeadingSize.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Heading from '../../lib/Heading';
+
+const HeadingSize = () => (
+  <div>
```

Comment:
Updated the nightwatch page: https://github.com/cerner/terra-core/pull/470/commits/286b3bc3f2998714cd1aa971c4c7120e9689b622

The https://terra-core-deployed-pr-470.herokuapp.com/static/#/site/heading page includes all the size examples as well.